apps/Node.js/credits

```diff
@@ -1,2 +1 @@
-Script by ilobilo
-Script improved by Itai-Nelken
+Script by AvilekkkDev
```

Comment:
```suggestion
Script by AvilekkkDev
Script by ilobilo
Script improved by Itai-Nelken
```
wow, thats kind of a .... move to remove someones name from the credits, because you changed a few numbers

Reply:
its like a full recoding... ye thats trash not code but works fine, the old one was completely broken